### DISCUSSION TITLE
Adding support for Nova custom blocks in addition to ItemsAdder and Oraxen ones

### DISCUSSION THREAD

**Discussion by Rinuuri** [BG-Software-LLC/SuperiorSkyblock2#1922] [Approved Feature Requests] [answered]
I wanna have support for custom blocks from the plugin called [Nova](https://hangar.papermc.io/xenondevs/Nova) in the same way as we have support for ItemsAdder for example, because currently nova blocks don't even correctly processed by protection - you couldn't break nova blocks on other players' islands but you could place them without any issues, I'm even mentioning that they aren't counting into island level and worth

**Comment by Lemvs** [comment, marked as answer]
Added in latest dev build.

@OmerBenGera you can mark this answered and close it.
